src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -8,12 +8,15 @@
 from xmltodict import parse
 from glob import glob
 
-qos_maps_model = ['DSCP_TO_TC_MAP_LIST',
-            'DOT1P_TO_TC_MAP_LIST',
-            'TC_TO_PRIORITY_GROUP_MAP_LIST',
-            'TC_TO_QUEUE_MAP_LIST',
-            'MAP_PFC_PRIORITY_TO_QUEUE_LIST',
-            'PFC_PRIORITY_TO_PRIORITY_GROUP_MAP_LIST']
+qos_maps_model = [
+    'DSCP_TO_TC_MAP_LIST',
```

Comment:
Any reason for adding DSCP_TO_TC_MAP_LIST here?

Reply:
I just reorg the indent here, and add `CABLE_LENGTH_LIST` to the list `qos_maps_model`.

Reply:
ok